doggo/quadruped.py

```
@@ -0,0 +1,744 @@
+from isaacsim.core.utils.rotations import euler_to_rot_matrix, quat_to_euler_angles, quat_to_rot_matrix
+from isaacsim.core.utils.stage import get_current_stage, set_stage_up_axis
+from isaacsim.core.utils.types import ArticulationAction
+from omni.isaac.nucleus import get_assets_root_path
+from omni.isaac.sensor import IMUSensor, LidarRtx
+from pxr import Usd, UsdGeom, UsdPhysics, Sdf
+#from omni.isaac.quadruped.robots import SpotFlatTerrainPolicy
+from isaacsim.robot.policy.examples.robots import SpotFlatTerrainPolicy
+
+
+# USD and related
+from pxr import Gf, UsdGeom
```

Ruff (0.14.3)

Redefinition of unused `UsdGeom` from line 40

Remove definition: `UsdGeom`

(F811)